src/autocomplete-creatable.tsx

```diff
@@ -33,8 +34,10 @@ export function AutoCompleteCreatable(props: AutoCompleteCreatableProps) {
   ).item;
 
   const queryExistsInList = filteredResults.some(i => i.value === query);
+  const showWhenEmpty = isEmpty(query) ? alwaysDisplay : true;
+  console.log("showWhenEmpty :>> ", showWhenEmpty);
```

Comment:
Just realized this console.log snuck into the release @anubra266

Reply:
Thanks will remove it in the next release